src/docs.md

```diff
@@ -10,12 +10,17 @@ Cadenza JS is a JavaScript library to use the [disy Cadenza](https://www.disy.ne
 
 Cadenza JS is included in the Cadenza distribution in the corresponding version.
 
-Alternatively you can install the most recent version using npm:
+Alternatively you can install the most recent version for a particular Cadenza Release using npm:
 
 ```bash
-npm install @disy/cadenza.js
+npm install @disy/cadenza.js@~10.2.0 # For latest version for Cadenza 10.2 
 ```
 
+The Cadenza Version is reflected in the cadenza.js version (e.g. 10.2.0 for Cadenza 10.2), while the last version segment is increased for both, bugfixes and functional changes.
```

Comment:
It's Cadenza JS throughout these docs.

Reply:
Fixed.

Reply:
Not all: "... corresponding major and minor version of cadenza.js"

Reply:
Changed.